### DISCUSSION TITLE
Textbox Width with Info

### DISCUSSION THREAD

**Discussion by mtanco** [h2oai/wave#879] [Ideas]
In a form, not every item would need additional information, but if the developer using tooltip for only some boxes the form item widths will change which looks a bit strange. It might be nice to not change the width or show the (i) in a way that does not need a shorter textbox

<img width="644" alt="Screen Shot 2021-06-14 at 7 53 51 AM" src="https://user-images.githubusercontent.com/4793561/121913192-07c7a180-cce6-11eb-86cf-1773c844906f.png">

**Comment by mturoci** [comment]
Tooltip icons should not affect any elements so that width would remain the same with or without the tooltip. I think we will address that as part of https://github.com/h2oai/wave/issues/326 in point:

> The implementation needs to be customized to each component, choosing a mechanism appropriate to the component.
